### PR TITLE
Add 'View in Kibana' Button [RHCLOUD-20313]

### PR DESCRIPTION
**Pull request by CodyWMitchell**
Added the ability to easily `View in Kibana` for a specific request_id on the `/track` page and filter by the service name by clicking `Filter in Kibana` from the details table. [RHCLOUD-20313](https://issues.redhat.com/browse/RHCLOUD-20313)

KibanaLink

### CODE REVIEW

File: deployment/deploy.yml [hunk 1]
```diff
@@ -67,6 +67,13 @@ objects:
             initialDelaySeconds: 30
             periodSeconds: 5
             timeoutSeconds: 2
+          env:
```
Comment (CodyWMitchell) [inline deployment/deploy.yml:70]: @SteveHNH I haven't added the environment variable overrides to app-interface, but is it looking right on this end?

Comment (CodyWMitchell) [reply deployment/deploy.yml]: Currently clowder environment-variables aren't supported for front-ends. This is being worked on as part of [RHCLOUD-21050](https://issues.redhat.com/browse/RHCLOUD-21050). In the meantime, I'm adding an additional API endpoint to handle this configuration [through this PR](https://github.com/RedHatInsights/payload-tracker-go/pull/115).

**Review by SteveHNH — APPROVED** [review]
Looks good to me!

**Review by aleccohan — APPROVED** [review]
Looks great to me!

**Review by tahmidefaz — APPROVED** [review]
Great stuff!

**Pull request merged**